src/screens/VaultScreen.js

```diff
@@ -37,22 +40,53 @@ export default class VaultScreen extends React.Component {
           upvotes: 0,
           category: 2,
           id: 1,
```

Comment:
is there a reason we are keeping this comment here? since the vault is now being populated, we no longer need this?

Reply:
true